src/ConcreteOperations/issubset.jl

```diff
@@ -221,10 +217,29 @@ compute a witness.
 Since ``S`` is convex, ``P ⊆ S`` iff ``v_i ∈ S`` for all vertices ``v_i`` of
 ``P``.
 """
-function ⊆(P::AbstractPolytope{N}, S::LazySet{N}, witness::Bool=false
-          )::Union{Bool, Tuple{Bool, Vector{N}}} where {N<:Real}
+function ⊆(P::AbstractPolytope{N}, S::LazySet{N}, witness::Bool=false;
+           algorithm=_default_issubset(P, S)) where {N<:Real}
     @assert dim(P) == dim(S)
 
+    if algorithm == "constraints"
+        return _issubset_constraints_list(P, S, witness)
+    elseif algorithm == "vertices"
+        return _issubset_vertices_list(P, S, witness)
+    else
+        error("algorithm $algorithm unknown")
+    end
+end
+
+@inline function _default_issubset(P, S)
+    if !applicable(constraints_list, S)
```

Comment:
I would typically not check for negated conditions (harder to read) in `if`-`else` blocks.